fuzz/Cargo.lock

```diff
@@ -589,18 +624,19 @@ dependencies = [
 
 [[package]]
 name = "hashbrown"
-version = "0.12.3"
+version = "0.14.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "8a9ee70c43aaf417c914396645a0fa852624801b24ebb7ae78fe8272889ac888"
+checksum = "2c6201b9ff9fd90a5a3bac2e56a830d0caa509576f0e503818ee82c181b3437a"
 dependencies = [
  "ahash",
+ "allocator-api2",
```

Comment:
This crate is optional according to https://crates.io/crates/hashbrown/0.14.0/dependencies , is it possible to avoid adding it?

Reply:
I didn't update it manually. That what `cargo update` produced. I don't see any command line options to exclude optional dependencies. Maybe we have to define required features for all dependency crates in parsec-fuzz cargo.toml or filter Cargo.lock manually.

Reply:
I can run `cargo update --package picky-asn1 --package picky-asn1-der --package picky-asn1-x509` instead of `cargo update` for this PR and leave running `cargo update` for the release process if you prefer

Reply:
I think it's better if we only update the necessary here and we update the rest for the release, as it would be better to avoid the issue of adding unneeded dependencies.